presto-hive/src/main/java/io/prestosql/plugin/hive/HdfsConfigurationInitializer.java

```diff
@@ -84,13 +85,15 @@ public HdfsConfigurationInitializer(HdfsConfig config, Set<ConfigurationInitiali
         this.configurationInitializers = ImmutableSet.copyOf(requireNonNull(configurationInitializers, "configurationInitializers is null"));
     }
 
-    private static Configuration readConfiguration(List<String> resourcePaths)
+    private static Configuration readConfiguration(List<File> resourcePaths)
     {
         Configuration result = new Configuration(false);
 
-        for (String resourcePath : resourcePaths) {
+        for (File resourcePath : resourcePaths) {
+            checkArgument(resourcePath.exists(), "File does not exist: " + resourcePath);
+
             Configuration resourceProperties = new Configuration(false);
-            resourceProperties.addResource(new Path(resourcePath));
+            resourceProperties.addResource(new Path(resourcePath.toURI()));
```

Comment:
am i correct this intentionally is not `resourcePath.toString()`?

Reply:
Thanks,let me update to use `toString`.